llvmlite/tests/test_binding.py

```diff
@@ -1382,6 +1546,35 @@ def test_run(self):
         self.assertIn("%.4", orig_asm)
         self.assertNotIn("%.4", opt_asm)
 
+    def test_run_with_remarks(self):
+        mod = self.module(licm_asm)
+        fn = mod.get_function("licm")
+        pm = self.pm(mod)
+        pm.add_licm_pass()
+        self.pmb().populate(pm)
+        mod.close()
+
+        pm.initialize()
+        (ok, remarks) = pm.run_with_remarks(fn)
+        pm.finalize()
+        self.assertTrue(ok)
+        self.assertIn("Passed", remarks)
+        self.assertIn("licm", remarks)
+
+    def test_run_with_remarks_filter(self):
+        mod = self.module(licm_asm)
+        fn = mod.get_function("licm")
+        pm = self.pm(mod)
+        pm.add_licm_pass()
+        self.pmb().populate(pm)
+        mod.close()
+
+        pm.initialize()
+        (ok, remarks) = pm.run_with_remarks(fn, remarks_filter="nothing")
```

Comment:
Maybe? The existing test case is already covering this since it's checking that the data matches an empty regular expression, which always passes.

Reply:
I suppose, however I think adding a test that checked in the positive case against a "real" value would give us confidence it's wired up suitably and not just working by accident via some strange empty string behaviour (I also know it's fine because I wrote the test locally!).